src/frontend/ai-reviewer-admin/src/domain/documents/DocumentTypeEditor.js

```diff
@@ -125,13 +125,11 @@ export default function DocumentTypeEditor() {
           )}
           <Button
             label="Cancel"
-            data-testid="cancel-config-btn"
```

Comment:
I think Karthik specifically added this line recently for his testing.  Are you sure you need to remove this?

https://github.com/bcgov/jag-file-submission/commit/9350ba7442c4d0728bc0ed820cb1605a280a1a78

Reply:
If it's OK, I'd like to keep the testid. Thanks!

Reply:
I'm not even sure why it was removed so all good